### DISCUSSION TITLE
Galaxy Watch 6 support

### DISCUSSION THREAD

**Discussion by Indever2** [HandyGram/app#7] [General]
Hi, guys! Just checked if it works on my Galaxy Watch 6 Classic. Some feedback:
1. Chats are grayed out at first launch. I got my chats after relaunching the app.
![image](https://github.com/HandyGram/HandyGram/assets/17948367/4346cea5-2576-4b07-ac44-da497b81ad40)

2. In my GW 6 Classic there's a default gesture to go back / close apllication - swipe from right to left to the edge of screen. In the app it does not work with all screens except the chat screen - i can go back to list of chats (this one is working):
![image](https://github.com/HandyGram/HandyGram/assets/17948367/0aa5b8f9-1a62-4644-8149-cdb002844b2d)

That's it i believe with major problems on this device. Hope it's helpful.
Maybe i'll create a PR if i have time to learn flutter and fix the bugs

P.S. Enable issues in the repo pls :)

**Comment by futurejohn** [comment]
I'm interested in getting involved with the development process too. I mostly do web development, but have been looking for a project to learn about mobile programming. 

Testing the app for bugs and getting a list of issues sounds like a good start. Is this a good place to talk about it or is there a Discord or something else?
